### PR TITLE
Change length of order slugs to 36 characters

### PR DESCRIPTION
**Pull request by maximdeclercq**
More secure URLs are desirable if Microsoft Auth gets merged (prevent people from bruteforcing the slug).

### CODE REVIEW

File: app/migrations/versions/3f76cdeb986d_.py
Comment (maximdeclercq) [inline app/migrations/versions/3f76cdeb986d_.py:1]: It seems like this migration was missing?

File: app/models/order.py [hunk 1]
```diff
@@ -3,21 +3,15 @@
...
 from .user import User
 
-BASE31_ALPHABET = '23456789abcdefghjkmnpqrstuvwxyz'
-
 def generate_slug():
-    secret = ''.join(secrets.choice(BASE31_ALPHABET) for i in range(8))
-    while Order.query.filter(Order.slug == secret).first() is not None:
-        secret = ''.join(secrets.choice(BASE31_ALPHABET) for i in range(8))
-    return secret
+    return secrets.token_urlsafe(24)
```
Comment (maximdeclercq) [inline app/models/order.py:14]: `token_urlsafe` generates 24 bytes and formats it as base64, which results in a character length of 32

File: app/migrations/versions/3f76cdeb986d_.py
Comment (maximdeclercq) [inline app/migrations/versions/3f76cdeb986d_.py:1]: It seems like this migration was missing?

File: app/models/order.py [hunk 2]
```diff
@@ -3,22 +3,21 @@
...
 from utils import first
 
 from .database import db
 from .user import User
 
-BASE31_ALPHABET = '23456789abcdefghjkmnpqrstuvwxyz'
 
 def generate_slug():
-    secret = ''.join(secrets.choice(BASE31_ALPHABET) for i in range(8))
+    secret = secrets.token_urlsafe(24)
```
Comment (maximdeclercq) [inline app/models/order.py:15]: Fyi token_urlsafe generates 24 bytes and formats it as base64, which results in a character length of 32, so the field is 32 chars.

**Comment by redfast00** [comment]
Ik heb hier nog eens over gesproken, 8 karakters was toch genoeg zodat dat niet online-bruteforcebaar is, en 36 is nogal onhandig om manueel over te typen.

**Pull request closed without merging**